Application/mobile/app_code/lib/main.dart

```diff
@@ -4,11 +4,12 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:flutter_stripe/flutter_stripe.dart';
 import 'package:provider/provider.dart';
 import 'package:risu/pages/home/home_page.dart';
+import 'package:risu/pages/login/login_refresh_token.dart';
 import 'package:risu/utils/providers/language.dart';
 import 'package:risu/utils/providers/theme.dart';
 import 'package:shared_preferences/shared_preferences.dart';
+
```

Comment:
espace en trop

Reply:
No, that's how the format is. Relative path must be separated by an empty line from absolute path